### DISCUSSION TITLE
Would a conda build of this be helpful to anyone?

### DISCUSSION THREAD

**Discussion by rla3rd** [dgunning/edgartools#169] [Ideas]
I have a working conda build from my local git fork of the repo.  I could add the conda environment yaml file and the meta.yaml file used for the build recipe to the base source directory.  Its pretty sparse as it has no tests, but it does work.  it could then be uploaded to the anaconda repository.

**Comment by dgunning** [comment]
This is a good suggestion. I don't use conda but other people might find it useful
